docs/docs/api/dbsp.md

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 3
+---
+
+# DBSP
+
+DBSP API on docs.rs
```

Comment:
Do we have to put a link here?

Reply:
Yes these all (except your SQL docs) are placeholders for now. We need to complete it.